2021-07-03 00:26:51,244	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 00:26:51,244	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 01:39:09,172	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 01:39:09,173	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-03 01:39:11,066	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 01:39:11,066	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 01:45:43,049	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 01:45:43,049	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 01:47:53,938	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 01:47:53,938	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /robots.txt
2021-07-03 02:31:58,224	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 02:31:58,224	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /owa/
2021-07-03 02:48:12,229	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 02:48:12,230	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-03 02:48:14,355	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 02:48:14,356	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 04:07:01,431	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 04:07:01,431	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 04:48:51,527	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: 'api.binance.com'. You may need to add 'api.binance.com' to ALLOWED_HOSTS.
2021-07-03 04:48:51,528	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/v1/time
2021-07-03 04:48:58,426	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 04:48:58,426	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 04:48:58,976	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 04:48:58,976	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 06:33:40,327	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:40,327	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/jsonws/invoke
2021-07-03 06:33:41,220	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:41,221	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 06:33:43,146	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:43,147	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /index.php
2021-07-03 06:33:44,529	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:44,530	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /mifs/.;/services/LogService
2021-07-03 06:33:45,500	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:45,500	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /wp-content/plugins/wp-file-manager/readme.txt
2021-07-03 06:33:46,413	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:46,413	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /console/
2021-07-03 06:33:48,988	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:48,988	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /Autodiscover/Autodiscover.xml
2021-07-03 06:33:49,959	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:49,960	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /_ignition/execute-solution
2021-07-03 06:33:50,887	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:50,888	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 06:33:53,515	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:53,515	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /vendor/phpunit/phpunit/src/Util/PHP/eval-stdin.php
2021-07-03 06:33:53,720	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:33:53,721	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /vendor/phpunit/phpunit/src/Util/PHP/eval-stdin.php
2021-07-03 06:51:30,769	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 06:51:30,770	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 07:01:36,028	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 07:01:36,029	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 07:44:43,472	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 07:44:43,472	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /ecp/Current/exporttool/microsoft.exchange.ediscovery.exporttool.application
2021-07-03 08:11:18,281	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 08:11:18,281	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 08:29:57,735	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '127.0.0.1:8000'. You may need to add '127.0.0.1' to ALLOWED_HOSTS.
2021-07-03 08:29:57,735	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 09:18:16,916	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 09:18:16,916	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 09:23:40,162	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 09:23:40,163	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 09:23:40,836	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 09:23:40,836	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 09:45:38,277	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 09:45:38,277	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 10:07:04,391	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 10:07:04,391	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /NwOI
2021-07-03 10:07:49,774	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 10:07:49,774	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-03 10:07:50,741	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 10:07:50,741	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 10:22:21,030	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 10:22:21,030	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 10:22:21,609	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 10:22:21,610	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 10:57:12,152	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 10:57:12,152	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /admin/config.php
2021-07-03 11:38:22,905	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 11:38:22,905	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 12:57:06,319	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 12:57:06,319	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 12:58:21,495	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 12:58:21,495	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 13:03:15,731	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 13:03:15,732	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 13:04:23,016	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 13:04:23,016	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /robots.txt
2021-07-03 13:04:24,618	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 13:04:24,619	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /sitemap.xml
2021-07-03 13:04:25,983	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 13:04:25,983	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.well-known/security.txt
2021-07-03 13:04:29,497	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 13:04:29,497	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /favicon.ico
2021-07-03 13:04:42,361	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 13:04:42,361	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 13:25:13,022	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 13:25:13,023	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 13:58:25,052	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 13:58:25,053	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 14:26:07,955	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 14:26:07,955	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /invoker/readonly
2021-07-03 14:26:09,056	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 14:26:09,057	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /_ignition/execute-solution
2021-07-03 14:26:10,124	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 14:26:10,124	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /vendor/phpunit/phpunit/src/Util/PHP/eval-stdin.php
2021-07-03 14:26:11,144	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 14:26:11,144	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 14:26:12,170	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 14:26:12,170	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /login
2021-07-03 14:26:13,231	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 14:26:13,231	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /jenkins/login
2021-07-03 14:26:14,212	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 14:26:14,212	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /manager/html
2021-07-03 14:26:15,569	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 14:26:15,569	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /wp-login.php
2021-07-03 14:26:16,921	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 14:26:16,921	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 15:09:54,749	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 15:09:54,749	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.git/config
2021-07-03 15:44:11,149	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 15:44:11,149	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 15:55:20,150	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: 'api.binance.com'. You may need to add 'api.binance.com' to ALLOWED_HOSTS.
2021-07-03 15:55:20,150	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/v1/time
2021-07-03 17:04:23,016	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 17:04:23,016	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 18:22:22,225	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 18:22:22,226	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /owa/auth/logon.aspx
2021-07-03 18:44:05,120	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 18:44:05,121	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-03 18:44:05,964	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 18:44:05,965	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 18:50:36,123	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 18:50:36,123	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 19:01:55,436	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:01:55,436	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 19:08:42,415	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: 'api.binance.com'. You may need to add 'api.binance.com' to ALLOWED_HOSTS.
2021-07-03 19:08:42,415	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/v1/time
2021-07-03 19:21:18,065	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:21:18,065	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /cgi-bin/config.exp
2021-07-03 19:42:24,249	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:24,249	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /vendor/phpunit/phpunit/src/Util/PHP/eval-stdin.php
2021-07-03 19:42:24,254	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:24,254	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 19:42:25,107	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:25,108	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /wp-content/plugins/wp-file-manager/readme.txt
2021-07-03 19:42:25,410	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:25,411	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /vendor/phpunit/phpunit/src/Util/PHP/eval-stdin.php
2021-07-03 19:42:26,662	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:26,663	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 19:42:26,760	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:26,760	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /console/
2021-07-03 19:42:28,278	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:28,279	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /Autodiscover/Autodiscover.xml
2021-07-03 19:42:28,327	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:28,328	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /_ignition/execute-solution
2021-07-03 19:42:29,199	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:29,199	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /index.php
2021-07-03 19:42:29,351	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:29,351	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /api/jsonws/invoke
2021-07-03 19:42:30,578	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 19:42:30,578	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /solr/admin/info/system
2021-07-03 20:56:13,831	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 20:56:13,831	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-03 20:56:14,327	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 20:56:14,327	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 20:56:17,482	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 20:56:17,482	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-03 20:56:18,318	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 20:56:18,318	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-03 21:42:35,834	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 21:42:35,834	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /actuator/health
2021-07-03 23:05:51,446	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 23:05:51,447	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /owa/
2021-07-03 23:45:08,858	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 23:45:08,858	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /.env
2021-07-03 23:45:09,499	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-03 23:45:09,499	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-04 00:00:35,749	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-04 00:00:35,749	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-04 00:05:38,772	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-04 00:05:38,773	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-04 00:07:49,296	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172:443'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-04 00:07:49,296	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-04 00:07:49,801	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-04 00:07:49,801	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-04 00:19:21,634	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-04 00:19:21,635	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
2021-07-04 00:19:22,619	[ERROR]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/core/handlers/exception.py(Line:101)	Invalid HTTP_HOST header: '54.248.148.172'. You may need to add '54.248.148.172' to ALLOWED_HOSTS.
2021-07-04 00:19:22,619	[WARNING]	/home/labolibrary_admin/venv_labolibrary/lib64/python3.7/site-packages/django/utils/log.py(Line:230)	Bad Request: /
